olive/passes/onnx/transformer_optimization.py

```diff
@@ -193,6 +205,18 @@ def _run_for_config(
             optimizer.convert_float_to_float16(
                 keep_io_types=config["keep_io_types"], op_block_list=op_block_list, force_fp16_inputs=force_fp16_inputs
             )
+            if config["use_gqa"]:
+                # Replace MultiHeadAttention with GroupQueryAttention and remove attention mask nodes
+                num_kv_heads = model.model_attributes.get("num_key_value_heads", None)
+                if num_kv_heads is None:
```

Comment:
Not a blocker: Should we make it possible to provide the value as a pass config parameter too?
I think there is an asymmetry that `num_head` and `hidden_size` can be pass config param or model attributes. 
Also, we don't usually provide model_attributes manually for non-automatic workflows.

Reply:
Yes, we can. But the GQA currently only works for llama-70B as the num_heads!=num_kv_heads. 
We can complete the hf_mapping to introduce similar value for num_kv_heads.